server/bin/gold/test-5.2.txt

```diff
@@ -968,31 +968,31 @@ run-1970-01-01T00:00:42-UTC: Processed 10 result tar balls, 7 successfully (0 pa
 1970-01-01T00:00:42.000000 DEBUG pbench-index.indexing_tarballs process_tb -- start processing list of tar balls
 1970-01-01T00:00:42.000000 INFO pbench-index.indexing_tarballs process_tb -- Starting /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/TO-INDEX/tarball-0_1970.01.01T00.42.00.tar.xz (size 212)
 1970-01-01T00:00:42.000000 DEBUG pbench-index.indexing_tarballs process_tb -- open tar ball
-1970-01-01T00:00:42.000000 WARNING pbench-index.indexing_tarballs process_tb -- Tar ball does not contain a metadata.log file: /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-0_1970.01.01T00.42.00.tar.xz - tar ball is missing "tarball-0_1970.01.01T00.42.00/metadata.log".
+1970-01-01T00:00:42.000000 WARNING pbench-index.indexing_tarballs emit_error -- Tar ball does not contain a metadata.log file: /var/tmp/pbench-test-server/test-5.2/pbench/archive/fs-version-001/controller-b-with-prefixes/tarball-0_1970.01.01T00.42.00.tar.xz - tar ball is missing "tarball-0_1970.01.01T00.42.00/metadata.log".
```

Comment:
We can take a poll @ndokos @dbutenhof @portante @npalaska

Reply:
It doesn't concern me

Reply:
Yes, this was the main point of my comment on the `emit_error` method; we're changing the logged method name; and while it's a visible difference, I'm not convinced it's worth the effort to fix. In theory, we could unwind the stack trace a level and log the caller frame rather than the current frame, but I don't see that added complication being of much value in this case.

I wouldn't want to elevate everything to exception, because it means everything becomes ERROR.

Reply:
Yeah, adding the stack unwinding thing seems like more effort and complication than is warranted for this situation.

Given that changing from warning to exception results in a change in semantics, we should probably go with what we have.  (Yeah, we could invent some additional mechanism, but, again, I don't think it's warranted.)